### ISSUE TITLE
Cannot set modifiers when a decorator exists

### ISSUE THREAD

**Issue by anthonybriand** [dsherret/ts-simple-ast#89]
Hello,

I use your module for TypeScript file generation from AST, but I have an error in two cases : 

**On a class :** 
If I declare decorators and set the isExported to true it throw the error : 
```
Error: Error replacing tree! Perhaps a syntax error was inserted (Current: ExportKeyword -- New: Decorator).

Code:
@HOInherit()
    at BaseError (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/errors/BaseError.js:5:9)
    at InvalidOperationError (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/errors/InvalidOperationError.js:6:9)
    at handleNode (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:57:19)
    at handleNode (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:65:17)
    at handleChangingParent (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:78:13)
    at handleNode (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:63:17)
    at handleNode (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:65:17)
    at replaceTree (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:53:9)
    at Object.replaceTreeCreatingSyntaxList (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:13:5)
    at Object.insertSyntaxList (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/insertion/insertSyntaxList.js:12:12)
```

**On a property :** 
If I declare decorators and set the scope to private it throw the error : 
```
Error: Error replacing tree! Perhaps a syntax error was inserted (Current: PrivateKeyword -- New: Decorator).

Code:
    @HOField()
    at BaseError (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/errors/BaseError.js:5:9)
    at InvalidOperationError (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/errors/InvalidOperationError.js:6:9)
    at handleNode (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:57:19)
    at handleNode (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:65:17)
    at handleChangingParent (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:78:13)
    at handleNode (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:63:17)
    at handleNode (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:65:17)
    at handleNode (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:65:17)
    at handleNode (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:65:17)
    at replaceTree (/Users/acidspike/Development/Sources/SCM/External/TSSimpleAst/dist/manipulation/tree/replaceTree.js:53:9)
```

**Comment by dsherret** [comment]
Thanks for reporting this @anthonybriand!

This is fixed in 0.70.1.

**Comment by anthonybriand** [comment]
Sorry, but errors are always here.

There is the code sample for reproduction : 
```
const sourceFile = ast.addSourceFileFromStructure('./test.ts', {
    imports: [
        {
            namedImports: [
                {
                    name: 'HOModel'
                },
                {
                    name: 'HOInherit'
                },
                {
                    name: 'HOMapping'
                },
                {
                    name: 'HOConnectionInfo'
                },
                {
                    name: 'HOField'
                }
            ],
            moduleSpecifier: '@harmony/orm'
        }
    ],
    classes: [
        {
            name: 'TestModel',
            'extends': 'HOModel',
            isExported: true,
            decorators: [
                {
                    name: 'HOInherit',
                    arguments: []
                },
                {
                    name: 'HOMapping',
                    arguments: [
                        '\'test\''
                    ]
                },
                {
                    name: 'HOConnectionInfo',
                    arguments: [
                        '\n    {\n        connectionName: \'test\'\n    }\n'
                    ]
                }
            ],
            properties: [
                {
                    name: 'test',
                    type: 'string',
                    decorators: [
                        {
                            name: 'HOField',
                            arguments: []
                        }
                    ],
                    scope: 'private',
                    initializer: 'null'
                }
            ],
            methods: [
                {
                    name: 'getTest',
                    bodyText: 'return this.test;',
                    returnType: 'string'
                },
                {
                    name: 'setTest',
                    parameters: [
                        {
                            name: 'test',
                            type: 'string'
                        }
                    ],
                    bodyText: 'this.test = test;\nreturn this;',
                    returnType: 'this'
                }
            ]
        }
    ]
});

console.log(sourceFile.getFullText()); // Throw errors
```
